malariagen_data/anoph/fst.py

```diff
@@ -105,10 +106,10 @@ def _fst_gwss(
     )
     def fst_gwss(
         self,
-        contig: base_params.contig,
-        window_size: fst_params.window_size,
-        cohort1_query: base_params.sample_query,
-        cohort2_query: base_params.sample_query,
+        region: Optional[base_params.region] = None,
+        window_size: Optional[fst_params.window_size] = None,
+        cohort1_query: Optional[base_params.sample_query] = None,
+        cohort2_query: Optional[base_params.sample_query] = None,
```

Comment:
Not sure why the type annotations of these parameters needs to change.

Reply:
It looks like I tried to explain the reason for this in [a comment above](https://github.com/malariagen/malariagen-data-python/pull/691#issuecomment-2631196701) but I should revisit this to double-check.